web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -756,6 +924,11 @@ const TeamReviews = ({ onBack, periodId }) => {
               >
                 <AddCircle />
               </IconButton>
+              {approvalMode && (
```

Comment:
An "APPROVE" or "UNAPPROVE" button only appears at the end of each member row when in approval mode.